File: app/src/main/java/com/situ2001/cliptokindle/bean/text/TextSetHelper.java
package com.situ2001.cliptokindle.bean.text;

import java.util.List;

public class TextSetHelper {
    //private static final TextSet textSet = new TextSet();
    private static final TextSet textSet = new TextSet();

    public static TextSet get() {
        return textSet;
    }

    public static List<Text> getList() {
        return textSet.getList();
    }
}

File: app/src/main/java/com/situ2001/cliptokindle/text/TextSetHelper.kt
package com.situ2001.cliptokindle.text

object TextSetHelper {
    //private static final TextSet textSet = new TextSet();
    private val textSet: TextSet = TextSet()
    fun get(): TextSet {
        return textSet
    }

    fun getList(): MutableList<Text> {
        return textSet.getList()
    }
}